src/data/faq.json

```diff
@@ -2171,66 +2171,304 @@
                 }
             ],
             "R": [
+                {
+                    "term": "RAT",
+                    "anchor": "rat",
+                    "description": "Short for Rapid Antigen Test. See <a href='#glossary_rapid_antigen_test'>Rapid Antigen Test</a>"
+                },
+                {
+                    "term": "Rapid Antigen Test",
+                    "anchor": "rapid_antigen_test",
+                    "description": "Rapid antigen tests are similar in procedure to the self-test, but are performed by trained personnel. A comparison of the two antigen test types has been provided by the RKI: <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Downloads/Flyer-Antigentests.pdf?__blob=publicationFile'>Link to the comparison from the RKI</a> <br></br>Not all rapid antigen tests are similarly reliable. For this reason, the Paul Ehrlich Institute has published so-called minimum criteria for rapid antigen tests: <a href='https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/mindestkriterien-sars-cov-2-antigentests-01-12-2020.pdf?__blob=publicationFile&v=7'>Link to minimum criteria</a> <br></br>A positive result from a rapid antigen test is not a final diagnosis and should be confirmed by a <a href='#glossary_pcr_test'>PCR test</a>."
+                },
+                {
+                    "term": "Rapid Test",
+                    "anchor": "rapid_test",
+                    "description": "See <a href='#glossary_rapid_antigen_test'>Rapid Antigen Test</a>"
+                },
+                {
+                    "term": "Rapid Test Profile",
+                    "anchor": "rapid_test_profile",
+                    "description": "Since version 2.2 of the Corona-Warn-App you are able to create a rapid test profile with your data such as first and last name, date of birth, address, phone number and e-mail address. Using a rapid test profile, you ensure that registration at a testing center is faster. The profile contains a QR code, which you can show at the testing center. However, you should still have your ID card ready."
+                },
                 {
                     "term": "Recovery Certificate",
                     "anchor": "recovery_certificate",
                     "description": "The certificate of recovery is a <a href='#glossary_digital_certificate'>digital certificate</a> issued for a person who has demonstrably recovered from COVID-19 and proved immunity by a PCR test."
+                },
+                {
+                    "term": "Release",
+                    "anchor": "release",
+                    "description": "Corona-Warn-App Version"
+                },
+                {
+                    "term": "Risk Encounter",
+                    "anchor": "risk_encounter",
+                    "description": "Encounter with a person who has potentially endangered oneself"

```

Comment:
Missing full stop at end of this sentence.

Reply:
Will be changed in the next commit, thanks!